sdk/monitor/azure-monitor-query/TROUBLESHOOTING.md

```diff
@@ -0,0 +1,193 @@
+* [Troubleshooting Metrics Query](#troubleshooting-metrics-query)
+    * [Troubleshooting insufficient access error](#troubleshooting-insufficient-access-error-for-metrics-query)
+    * [Troubleshooting unsupported granularity for metrics query](#troubleshooting-unsupported-granularity-for-metrics-query)
+* [Additional azure-core configurations](#additional-azure-core-configurations)
+
+## General Troubleshooting
+
+Monitor query raises exceptions described in [`azure-core`](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/core/azure-core/README.md)
+
+### Enable client logging
+
+To troubleshoot issues with Azure Monitor query library, it is important to first enable logging to monitor the behavior of the application. The errors and warnings in the logs generally provide useful insights into what went wrong and sometimes include corrective actions to fix issues.
+
+This library uses the standard [logging](https://docs.python.org/3/library/logging.html) library for logging. Basic information about HTTP sessions, such as URLs and headers, is logged at the INFO level.
+Detailed DEBUG level logging, including request/response bodies and unredacted headers, can be enabled on a client with the logging_enable argument:
+
+```python
+import logging
+from azure.monitor.query import LogsQueryClient
+
+# Create a logger for the 'azure.monitor.query' SDK
+logger = logging.getLogger('azure.monitor.query')
+logger.setLevel(logging.DEBUG)
+
+# Configure a console output
+handler = logging.StreamHandler(stream=sys.stdout)
+logger.addHandler(handler)
+
+client = LogsQueryClient(credential, logging_enable=True)
+```
+
+Similarly, logging_enable can enable detailed logging for a single operation, even when it isn't enabled for the client:
+
+```python
+client.query_workspace(logging_enable=True)
+```
+
+### Authentication errors
+
+Azure Monitor Query supports Azure Active Directory authentication. Both LogsQueryClient and
+MetricsQueryClient have methods to set the `credential`. To provide a valid credential, you can use
+`azure-identity` dependency. For more details on getting started, refer to
+the [README](https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/monitor/azure-monitor-query#create-the-client)
+of Azure Monitor Query library. You can also refer to
+the [Azure Identity documentation](https://docs.microsoft.com/python/api/overview/azure/identity-readme)
+for more details on the various types of credential supported in `azure-identity`.
+
+### Errors with running async APIs
+
+The async transport is designed to be opt-in. [AioHttp](https://pypi.org/project/aiohttp/) is one of the supported implementations of async transport. It is not installed by default. You need to install it separately.
+
+## Troubleshooting Logs Query
+
+### Troubleshooting insufficient access error for logs query
+
+If you get an HTTP error with status code 403 (Forbidden), it means that the provided credentials does not have
```

Comment:
How about adding a sentence at the bottom of this Auth section about "For more help on troubleshooting authentication errors please see the Azure Identity client library troubleshooting guide (with a link)"